docs/data/stashcache/run-stashcache-container.md

```diff
@@ -0,0 +1,195 @@
+
+[Install]
+WantedBy=multi-user.target
+```
+
+This systemd file can be saved to `/etc/systemd/system/docker.stash-cache.service` and started with:
+
+```console
+root@host $ systemctl enable docker.stash-cache
+root@host $ systemctl start docker.stash-cache
+```
+
+!!! warning
+    You must [register](/stashcache/install-cache/#registering-the-cache) the cache before considering it a production service.
+
+Additional Configuration (Optional)
+-----------------------------------
+
+In addition to the required configuration above, you may also configure the behavior of your cache with the following variables in your environment file:
+
+- `XC_SPACE_HIGH_WM`: High watermark for disk usage;
+      when usage goes above the high watermark, the cache deletes until it hits the low watermark
+- `XC_SPACE_LOW_WM`: Low watermark for disk usage;
+      when usage goes above the high watermark, the cache deletes until it hits the low watermark
+- `XC_PORT`: TCP port that XCache listens on
+- `XC_RAMSIZE`: Amount of memory to use for storing blocks before writting them to disk. (Use higher for slower disks).
+- `XC_BLOCKSIZE`: The size of the blocks in the cache
+- `XC_PREFETCH`: Number of blocks to prefetch from a file at once. 
+       This is a value of how aggressive the cache is to request portions of a file. Set to `0` to disable
+
+### Optimizing Stashcache ###
+
+#### Network Optimization ####
+
+For caches that are connected to NIC's over `40Gbps` we recommend to disable the virtualized network and "bind" the container to the host network:
+
+```console
+user@host $ docker run --rm  \
+             --network="host" \
+             --volume <HOST PARTITION>:/cache \
+             --env-file=/opt/xcache/.env \
+             opensciencegrid/stash-cache:stable
+```
+
+#### Memory Optimization ####
+
+Stashcache uses the hosts memory in two ways:
+
+1. Uses the own linux kernel as a way to cache files read from disk
+1. As a buffer for writting blocks of files first in memory and then in disk (to account for slow disks).
+
+An easy way to increase the performance of stashcache is to assign it more memory. You can use the [docker option](https://docs.docker.com/config/containers/resource_constraints/#limit-a-containers-access-to-memory) `--memory` and set it up to at least twice of `XC_RAMSIZE`.
+
+```console
+user@host $ docker run --rm --publish <HOST PORT>:8000 \
+             --memory=64g \
+             --volume <HOST PARTITION>:/cache \
+             --env-file=/opt/xcache/.env \
+             opensciencegrid/stash-cache:stable
+```
+
+
+#### Multiple Disks ####
+
+For caches that store over `10 TB` or that have assigned space for storing the cached files over multiple partitions (`/partition1, /partition2, ...`) we recommend the following.
+
+1. Create a config file `90-my-stash-cache-disks.cfg` with the following contents
```

Comment:
```suggestion
1. Create a config file `90-my-stash-cache-disks.cfg` on the host with the following contents:
```

But also, why don't we have them use the [cache-disks.config](https://github.com/opensciencegrid/docker-xcache/blob/master/xcache/image-config.d/10-disks-xcache.sh#L3-L9) feature?